<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QMenuBar" name="menubar">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>26</height>
       </rect>
      </property>
      <widget class="QMenu" name="menuFile">
       <property name="title">
        <string>Файл</string>
       </property>
       <addaction name="actionOpen"/>
       <addaction name="actionSave"/>
       <addaction name="actionExit"/>
      </widget>
      <widget class="QMenu" name="menuOptions">
       <property name="title">
        <string>Опции</string>
       </property>
       <addaction name="actionSettings"/>
      </widget>
      <widget class="QMenu" name="menuHelp">
       <property name="title">
        <string>Справка</string>
       </property>
       <addaction name="actionAbout"/>
       <addaction name="actionManual"/>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QToolBar" name="toolBar">
      <attribute name="toolBarArea">
       <enum>TopToolBarArea</enum>
      </attribute>
      <attribute name="toolBarBreak">
       <bool>false</bool>
      </attribute>
      <attribute name="orientation">
       <enum>Horizontal</enum>
      </attribute>
      <attribute name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </attribute>
      <attribute name="toolButtonStyle">
       <enum>ToolButtonTextUnderIcon</enum>
      </attribute>
      <addaction name="actionAddRecord"/>
      <addaction name="actionAnalysis"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="actionManual">
   <property name="text">
    <string>Инструкция пользователя</string>
   </property>
  </action>
  <action name="actionAddRecord">
   <property name="text">
    <string>Добавить запись</string>
   </property>
  </action>
  <action name="actionAnalysis">
   <property name="text">
    <string>Анализ</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWidget</class>
   <extends>QWidget</extends>
   <header>QWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
